secboot/encrypt.go

```diff
@@ -116,3 +116,12 @@ func NewAuxKey() (AuxKey, error) {
 	// On return, n == len(b) if and only if err == nil
 	return key, err
 }
+
+// EncryptionType specifies what encryption backend should be used (if any)
+type EncryptionType string
+
+const (
+	EncryptionTypeNone            EncryptionType = ""
+	EncryptionTypeCryptsetup      EncryptionType = "cryptsetup"
```

Comment:
Hm, maybe this should be "LUKS" instead of "cryptsetup". What does @chrisccoulson  thing?

Reply:
Samuele suggested to use LUKS so I will go with that